src/main/java/fr/techad/edc/popover/internal/swing/components/IconButtonListener.java

```diff
@@ -66,7 +67,27 @@ public void mousePressed(MouseEvent e) {
     @Override
     public void mouseReleased(MouseEvent e) {
         if (this.helpConfiguration.getSummaryDisplay()) {
-            openPopover(e.getXOnScreen(), e.getYOnScreen());
+            if(helpConfiguration.getPopoverPlacement() == null){
+                openPopover(e.getXOnScreen(), e.getYOnScreen());
+            }else{
+                switch(helpConfiguration.getPopoverPlacement()){
+                    case LEFT:
```

Comment:
if the popover size changes, your translation will be wrong. Define the placement in the popover and it will compute its position according to the click location

In openPopover, add the placement.
And call the location with it: popover.setLocation(x, y, **placement**);

Reply:
i use popoverPlacement import instead of put directly in parameter method of setLocation because that method from Window.java class.